lib/slax_web/websockets/default_repo.ex

```diff
@@ -68,13 +68,18 @@ defmodule SlaxWeb.DefaultRepo do
           type: "input",
           block_id: "channel",
           element: %{
-            type: "channels_select",
+            type: "multi_conversations_select",
             placeholder: %{
               type: "plain_text",
               text: "Select channel",
               emoji: true
             },
-            action_id: "channels_select_action"
+            action_id: "channels_select_action",
+            filter: %{
+              include: ["public", "private"],
+              exclude_external_shared_channels: true,
```

Comment:
so we could set this to true and itd work for vetavize?

Reply:
perhaps - this is what populates the channel dropdown when we're setting default repos for channels. Maybe I should set it to false; because there's no guarantee all the other stuff would work for an externally shared channel. 

IIRC, we'd have to add Slax to that workspace, and then there might be some shenanigans involving the oauth permissions for external channels.

TL;DR: ...maybe? 😹

Reply:
I'm changing it to false

Reply:
word nbd ty for explain